src/controller/index.ts

```diff
@@ -166,30 +168,39 @@ export default class SimulariumController {
     }
 
     public configureNetwork(config: NetConnectionParams): void {
-        if (this.simulator && this.simulator.socketIsValid()) {
-            this.simulator.disconnect();
+        if (this.simulator) {
+            this.simulator.abort();
         }
 
         this.createSimulatorConnection(config);
     }
 
+    public isRemoteOctopusClientIsConfigured(): boolean {
```

Comment:
I think this should be either `isRemoteOctopusClientConfigured` or `remoteOctopusClientIsConfigured`